src/util/error.js

```diff
@@ -1,11 +1,3 @@
-import { isUndefined } from './is-type';
-import root from 'window-or-global';
-
 export default function error (message) {
-  const { DEBUG } = root;
-  if (!isUndefined(DEBUG) && DEBUG) {
-    console.error(message);
-  } else {
-    throw new Error(message);
-  }
+  throw new Error(message);
```

Comment:
This will always throw. 
I removed the condition and the DEBUG flag. 
It will never console.error(), it always throws now.